web/models/character.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Yggdrasil.Character do
+  use Yggdrasil.Web, :model
+
+  schema "characters" do
+    field :name, :string
+    belongs_to :user, Yggdrasil.User
+    belongs_to :game, Yggdrasil.Game
+
+    timestamps
+  end
+
+  @required_fields ~w(name user_id game_id)
```

Comment:
I see these are required but I didn't see any tests documenting that. Also It looks like you can work with a `user` and `game` field which can contain the struct. Is there a reason all your code refers to the *_id fields and not the those?